aemedge/blocks/when-to-use/when-to-use.css

```diff
@@ -0,0 +1,136 @@
+.when-to-use-container {
+  margin: var(--udexSpacer40) 0;
+  
+  .default-content-wrapper {
+    h1 {
```

Comment:
is this something to apply only for the "when to use" section, or should this be applied for the whole template?

Reply:
I have it on my internal todo list and will check this out in the current issue #514 I am working on, if we should move general styles to the template.